support-frontend/assets/components/test-user-banner/testUserBanner.tsx

```diff
@@ -1,22 +1,29 @@
 import { css } from '@emotion/react';
 import { error, neutral, space, textSans } from '@guardian/source-foundations';
+import { useLocation } from 'react-router-dom';
 import { isTestUser } from 'helpers/user/user';
+import { ThankYouUserTypeSelector } from './thankYouUserTypeSelector';
 
 const testUserBannerStyles = css`
 	${textSans.large()};
 	background-color: ${error[400]};
 	color: ${neutral[100]};
 	text-align: center;
 	padding: ${space[2]}px 0;
+	font-weight: bold;
 `;
 
 export function TestUserBanner(): JSX.Element | null {
 	const testUser = isTestUser();
+	const location = useLocation();
```

Comment:
We could move the variable definitions for `location` and `isThankYouPage` inside the `if (testUser)` block, as we only use these 2 variables if this conditional is `true`.

Reply:
If `useLocation` is a hook then it can't be used inside a conditional block; hooks won't work properly if they're not called on every render.

Reply:
`useLocation` is a hook so have moved this out of the conditional block